### PR TITLE
Created a page for the Stepper component

### PR DESCRIPTION
**Pull request by imincheva**

### CODE REVIEW

File: en/components/stepper.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Use the Stepper Component to allow the user to visualize content as a process, which shows its progress by dividing the content into successive steps. The Stepper is visually identical to the [Ignite UI for Angular Stepper Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/stepper.html)
+
+## Stepper Demo
+
+<img class="responsive-img" src="../images/stepper_demo.png" srcset="../images/stepper_demo@2x.png 2x" />
+
+## Type
+
+The Stepper component supports two types - Linear and Non Linear. In Figma you can switch between the two using the "Type" property from the properties panel. Non Linear type is used to prevent the users from proceeding with the next step, without firstly completing the previous, non-optional ones.
```
Comment (AnjiManova) [inline en/components/stepper.md:17]: Non Linear -> Linear type is used to prevent ...

File: en/components/stepper.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Steps
+
+Each Step consists of Indicator and Information. The Indicator comes with two types - Number and Icon. You can switch between the two using the "Type" property from the properties panel. You can also choose whether you want to display only Indicator or Information. To do so, you can simply hide them from the layers panel. You also have the option to show/hide the Subtitle by swithing on/off the Subtitle boolean property of the nested Step component.
+
+<img class="responsive-img" src="../images/step_number.png" srcset="../images/step_number@2x.png 2x" />
+<img class="responsive-img" src="../images/step_icon.png" srcset="../images/step_icon@2x.png 2x" />
+
+We also provide a number of states for the Step, which should cover all use case scenarios. These include: Complete, Incomplete, Disabled, Optional, Error and Current. These can be switched from the "State" property on the properties panel. We also provide two additional states: Hover and Focused, which can be used by switching on/off the boolean properties "Hover" and "Focused".
```
Comment (AnjiManova) [inline en/components/stepper.md:43]: Error and Current -> Invalid and Active

File: en/components/stepper.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+We also provide two types of Progress Lines - Active and Inactive. These can be swapped by using the boolean property "Value" from the properties panel, after selecting the nested Progress Line component.
+
+<img class="responsive-img" src="../images/stepper_progress-line_active.png" srcset="../images/stepper_progress-line_active@2x.png 2x" />
+<img class="responsive-img" src="../images/stepper_progress-line_inactive.png" srcset="../images/stepper_progress-line_inactive@2x.png 2x" />
+
+## Content
+
+As mentioned above, When a Step is set to "Current" state, a frame called "Content" is displayed underneath the current Step on the vertical Stepper and underneath all steps on the horizontal Stepper. By default, the Content comes with a simple text layer, where you can add information about the current step. If you'd like to cusomize it in some way, you'll need to detach the component by right-clicking on it and selecting "Detach instance". You can also fully hide it by using the "Content" property on the horizontal Stepper or by going into the nested Step component and switching off the boolean property "Body" on the vertical Stepper.
```
Comment (AnjiManova) [inline en/components/stepper.md:55]: "Current" -> Active 
Underneath the Header within the current Step?

File: en/components/stepper.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+## Styling 
+
+The Stepper comes with styling flexibility through the various options for its indicators, texts and progress line colors, as well as through the customization of the different states' look.
+
+<img class="responsive-img" src="../images/stepper_styling.png" srcset="../images/stepper_styling@2x.png 2x" />
+
+## Usage 
+
+When using the linear Stepper, make sure you set the incompleted steps to "Disabled" state, as well as the progress lines to "Inactive" in order to tell the user he may not proceed with the next steps, until the previos ones are complete. You should avoid combining indicators with icons and indicators with numbers. Keep the current step promiment, in order to give the user adequate feedback on where he is within the process.
```
Comment (AnjiManova) [inline en/components/stepper.md:67]: the user he may -> the user they may
previos -> previous

File: en/components/stepper.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+<img class="responsive-img" src="../images/stepper_styling.png" srcset="../images/stepper_styling@2x.png 2x" />
+
+## Usage 
+
+When using the linear Stepper, make sure you set the incompleted steps to "Disabled" state, as well as the progress lines to "Inactive" in order to tell the user he may not proceed with the next steps, until the previos ones are complete. You should avoid combining indicators with icons and indicators with numbers. Keep the current step promiment, in order to give the user adequate feedback on where he is within the process. 
+
+| Do                                                                                 | Don't                                                                                  |
+| ---------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| <img class="responsive-img" src="../images/stepper_do1.png" srcset="../images/stepper_do1@2x.png 2x" /> | <img class="responsive-img" src="../images/stepper_dont1.png" srcset="../images/stepper_dont1@2x.png 2x" /> |
```
Comment (AnjiManova) [inline en/components/stepper.md:71]: swap do1 and don't1 images

File: en/components/stepper.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+---
+title: Stepper - Design System Component
+_description: The Stepper Component Symbol conveys progress through numbered steps.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits, Figma, Figma to Angular, Export code from Figma, Figma HTML, Figma to HTML, Figma UI kits
```
Comment (AnjiManova) [inline en/components/stepper.md:4]: Consider removing the Sketch keywords because we are not supporting the Stepper in Sketch

**Review by AnjiManova — CHANGES_REQUESTED** [review]
I left a couple of comments for review

File: en/components/stepper.md (hunk 1 above)
Comment (imincheva) [reply en/components/stepper.md:17]: Fixed.

File: en/components/stepper.md (hunk 2 above)
Comment (imincheva) [reply en/components/stepper.md:43]: Fixed.

File: en/components/stepper.md (hunk 3 above)
Comment (imincheva) [reply en/components/stepper.md:55]: Fixed.

File: en/components/stepper.md (hunk 4 above)
Comment (imincheva) [reply en/components/stepper.md:67]: fixed.

File: en/components/stepper.md (hunk 6 above)
Comment (imincheva) [reply en/components/stepper.md:4]: @sdimchevski should I do this? We were also wondering whether I should add notes on the new component pages, that currently we don't support the component in Sketch and Adobe XD.

Comment (sdimchevski) [reply en/components/stepper.md:4]: We can say the Stepper is coming to Sketch in the near future and leave out the Sketch keywords for now

File: en/components/stepper.md (hunk 5 above)
Comment (imincheva) [reply en/components/stepper.md:71]: Fixed

File: en/components/stepper.md (hunk 6 above)
Comment (imincheva) [reply en/components/stepper.md]: Fixed

**Review by AnjiManova — APPROVED** [review]

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**